### DISCUSSION TITLE
建议增加自动内存优化

### DISCUSSION THREAD

**Discussion by CHEN-CHEN-X** [Meloong-Git/PCL#3837] [提问] [answered]
可以设置超过多少清理一下或者定时清理，可以放启动前清理内存下面做一个拖动条

类似这样

![屏幕截图 2024-05-17 192955](https://github.com/Hex-Dragon/PCL2/assets/153986378/12943ce0-46cb-4d9d-b30c-3babcc3da3d3)

**Comment by WForst-Breeze** [comment, marked as answer]
#2414 #2221

**Reply by CHEN-CHEN-X** [reply]
好吧已经有一个类似的我问题了没看到
